data/serializer/src/main/java/tech/pegasys/teku/api/schema/altair/SyncCommitteeSignature.java

```diff
@@ -0,0 +1,86 @@
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.teku.api.schema.altair;
+
+import static tech.pegasys.teku.api.schema.SchemaConstants.DESCRIPTION_BYTES32;
+import static tech.pegasys.teku.api.schema.SchemaConstants.DESCRIPTION_BYTES96;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.swagger.v3.oas.annotations.media.Schema;
+import java.util.Objects;
+import org.apache.tuweni.bytes.Bytes32;
+import tech.pegasys.teku.api.schema.BLSSignature;
+import tech.pegasys.teku.infrastructure.unsigned.UInt64;
+import tech.pegasys.teku.spec.datastructures.operations.versions.altair.SyncCommitteeSignatureSchema;
+import tech.pegasys.teku.spec.datastructures.type.SszSignature;
+import tech.pegasys.teku.ssz.primitive.SszBytes32;
+import tech.pegasys.teku.ssz.primitive.SszUInt64;
+
+public class SyncCommitteeSignature {
+  @Schema(type = "string", format = "uint64")
+  @JsonProperty("slot")
+  public final UInt64 slot;
+
+  @Schema(type = "string", format = "byte", description = DESCRIPTION_BYTES32)
+  @JsonProperty("beacon_block_root")
+  public final Bytes32 beaconBlockRoot;
+
+  @Schema(type = "string", format = "uint64")
+  @JsonProperty("validator_index")
+  public final UInt64 validatorIndex;
+
+  @Schema(type = "string", format = "byte", description = DESCRIPTION_BYTES96)
+  @JsonProperty("signature")
+  public final BLSSignature signature;
+
+  @JsonCreator
+  public SyncCommitteeSignature(
+      @JsonProperty("slot") final UInt64 slot,
+      @JsonProperty("beacon_block_root") final Bytes32 beaconBlockRoot,
+      @JsonProperty("validator_index") final UInt64 validatorIndex,
+      @JsonProperty("signature") final BLSSignature signature) {
+    this.slot = slot;
+    this.beaconBlockRoot = beaconBlockRoot;
+    this.validatorIndex = validatorIndex;
+    this.signature = signature;
+  }
+
+  @Override
+  public boolean equals(final Object o) {
+    if (this == o) return true;
+    if (o == null || getClass() != o.getClass()) return false;
+    final SyncCommitteeSignature that = (SyncCommitteeSignature) o;
+    return Objects.equals(slot, that.slot)
+        && Objects.equals(beaconBlockRoot, that.beaconBlockRoot)
+        && Objects.equals(validatorIndex, that.validatorIndex)
+        && Objects.equals(signature, that.signature);
+  }
+
+  @Override
+  public int hashCode() {
+    return Objects.hash(slot, beaconBlockRoot, validatorIndex, signature);
+  }
+
+  public tech.pegasys.teku.spec.datastructures.operations.versions.altair.SyncCommitteeSignature
+      asInternalCommitteeSignature() {
+    return new tech.pegasys.teku.spec.datastructures.operations.versions.altair
+        .SyncCommitteeSignature(
+        SyncCommitteeSignatureSchema.INSTANCE,
+        SszUInt64.of(slot),
+        SszBytes32.of(beaconBlockRoot),
+        SszUInt64.of(validatorIndex),
+        new SszSignature(signature.asInternalBLSSignature()));
```

Comment:
hmm, this is somewhat unfortunate.  Ideally we'd go through `spec.getSchemaDefinitions` to create this.  Otherwise we're just hard coding specific schemas in more places.  I'm not 100% sure what the right trade off here is - specifically we need to handle the case where the slot is actually in phase0 and we can't create an internal version properly.